tests/ecosystem/deployment/test_rdr_latency_setup.py

```diff
@@ -0,0 +1,11 @@
+from ocs_ci.deployment.multicluster_deployment import setup_rdr_latency
+from ocs_ci.framework.pytest_customization.marks import rdr_latency
+
+####################################################################################################
+# This file is placeholder for setting up rdr cluster latency #
```

Comment:
@prsurve shouldn't we set up the latency as part of a setup fixture, at a session scope?